src/test/java/com/amazon/ion/impl/bin/IonRawBinaryWriterTest_1_1.kt

```diff
@@ -798,4 +808,212 @@ class IonRawBinaryWriterTest_1_1 {
             writeBool(false)
         }
     }
+
+    @Test
+    fun `write int`() {
```

Comment:
🗺️ For all of these scalar types, I'm relying on the tests for `IonEncoder_1_1` to ensure edge cases are covered since the implementations in `IonRawBinaryWriter_1_1` are basically pass through methods.